src/client/containers/CategoryPage/CategoryPage.Container.js

```diff
@@ -1,10 +1,24 @@
 import React from 'react';
 import './CategoryPage.Style.css';
+import ProductView from '../../components/ProductView/ProductView.component';
+import { useParams } from 'react-router-dom';
+import { useFetchApi } from '../../hooks/UseFetchApi';
 
 const CategoryPageContainer = () => {
+  const { name } = useParams();
+  const products = useFetchApi(`products?category=${name}`).data;
   return (
     <div>
-      <h1>Category page Container</h1>
+      {products.isLoading ? (
+        <h2>Loading...</h2>
```

Comment:
You can use Loader actualy, it is already created
`import Loader from '../../components/Loader/index';`
its good to reuse it on all ages because we can later change it to anymated loader for example and not rewrite code on each page

Reply:
done